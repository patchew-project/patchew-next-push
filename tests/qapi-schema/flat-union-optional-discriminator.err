tests/qapi-schema/flat-union-optional-discriminator.qapi:6: Discriminator of flat union 'MyUnion' does not allow optional name '*switch'
