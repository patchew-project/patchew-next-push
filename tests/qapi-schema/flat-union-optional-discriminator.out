object q_empty
enum QType ['none', 'qnull', 'qnum', 'qstring', 'qdict', 'qlist', 'qbool']
    prefix QTYPE
module flat-union-optional-discriminator.json
enum Enum ['one', 'two']
object Base
    member switch: Enum optional=True
object Branch
    member name: str optional=False
object MyUnion
    base Base
    tag switch
    default variant: one
    case one: Branch
    case two: Branch
